store/src/store.rs

```diff
@@ -188,7 +188,7 @@ pub trait ChainStore<'a>: Send + Sync {
     }
 
     fn is_main_chain(&'a self, hash: &packed::Byte32) -> bool {
-        self.get_block_number(&hash).is_some()
+        self.get(COLUMN_INDEX, hash.as_slice()).is_some()
```

Comment:
What is the difference between `self.get_block_number(&hash).is_some()` and `self.get(COLUMN_INDEX, hash.as_slice()).is_some()`?

Reply:
`get_block_number` will do deserialize which is redundant for this function